project/profiles/capacity4more/modules/c4m/demo/c4m_demo_content/includes/CreateOgContent.inc

```diff
@@ -90,6 +90,9 @@ abstract class C4mDemoContentCreateOgContent extends C4mDemoContentCreateEntity
 
     // Make sure the update message also exists.
     $this->messageUpdateSave($entity, $row);
+
+    // Update membership totals fields.
+    c4m_og_node_insert($entity);
```

Comment:
Ok. Thanks.

I saw it didn't work when I reinstalled yesterday.
But it might have been related to me using `global $user`:
![image](https://cloud.githubusercontent.com/assets/17882245/23958786/d92f28ea-09ab-11e7-9056-a77a88265a71.png)

Anyway, I will test migration again, without the hooks, and check...